packages/labs/ssr/package.json

```diff
@@ -33,38 +33,39 @@
   },
   "devDependencies": {
     "@koa/router": "^10.1.1",
-    "@open-wc/testing-karma": "^4.0.9",
     "@open-wc/testing": "^3.0.0-next.1",
+    "@open-wc/testing-karma": "^4.0.9",
     "@types/chai": "^4.2.11",
     "@types/command-line-args": "^5.0.0",
-    "@types/koa__router": "^8.0.2",
+    "@types/koa": "^2.0.49",
     "@types/koa-cors": "*",
     "@types/koa-static": "^4.0.1",
-    "@types/koa": "^2.0.49",
+    "@types/koa__router": "^8.0.2",
     "@types/mocha": "^9.0.0",
-    "@types/node-fetch": "^2.5.7",
     "@types/node": "^16.7.8",
```

Comment:
Just noticed we also have this in `devDependencies`. We should remove from here and keep the one in `dependencies`?

Reply:
Good catch, thanks. Removed.